index.js

```diff
@@ -0,0 +1,138 @@
+const URL = 'https://ada-backtrek-api.herokuapp.com/trips';
+
+
+const loadTrips = () => {
+
+  const tripList = $('#trip-list');
+
+  axios.get(URL)
```

Comment:
You probably want to clear the trip list first. As is, if you click the `load trips` button twice, the list appears two times.